task/keyaction.c

```diff
@@ -427,6 +427,11 @@ void KeypressAction(uint8_t Action) {
 				MENU_DrawSetting();
 				break;
 
+			case ACTION_BANDWIDTH:
+				gMenuIndex = MENU_BAND_WIDTH;
+				DISPLAY_Fill(0, 159, 1, 81, COLOR_BACKGROUND);
+				MENU_DrawSetting();
```

Comment:
Hi @jcalado,
Thanks for your proposition :fire:

The idea would be to switch directly between Narrow and Wide without opening the menu.
(I haven't tested my code suggestion, you might also have to redraw the W/N indicator to update display).
```suggestion
				gVfoState[gSettings.CurrentVfo].bIsNarrow ^= 1;
```

Reply:
my bad, I misinterpreted the issue. I'll have a look into it

Reply:
Switched it over to a toggle and updated the display. Thanks for the tip!